sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificatePolicy.cs

```diff
@@ -46,9 +57,29 @@ public class CertificatePolicy : IJsonSerializable, IJsonDeserializable
         private static readonly JsonEncodedText s_certificateTransparencyPropertyNameNameBytes = JsonEncodedText.Encode(CertificateTransparencyPropertyName);
 
         /// <summary>
-        /// The properties of the key backing a certificate
+        /// The type of backing key to be generated when issuing new certificates
         /// </summary>
-        public KeyOptions KeyOptions { get; set; }
+        public CertificateKeyType? KeyType { get; set; }
+
+        /// <summary>
+        /// Specifies whether the certificate key should be reused when rotating the certificate
+        /// </summary>
+        public bool? ReuseKey { get; set; }
+
+        /// <summary>
+        /// Specifies whether the certificate key is exportable from the vault or secure certificate store
+        /// </summary>
+        public bool? Exportable { get; set; }
+
+        /// <summary>
+        /// The curve which back the EC key
+        /// </summary>
+        public CertificateKeyCurveName? CurveName { get; set; }
```

Comment:
Should this be KeyCurveName?

Reply:
The type, no: we wanted it to be unambiguous with the Keys package. But if you mean the property name, yeah, I could see that. All but `Exportable` from the old `KeyOptions` is prefaced with "Key"; however, that would be inconsistent with `JsonWebKey` which has `KeyType` but `CurveName`, though one could argue maybe that should just be `Type`.